installer.sh

```diff
@@ -95,6 +95,9 @@ wget https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-2.0.0.tar.gz
 tar xzvf SDL2_mixer-2.0.0.tar.gz
 cd SDL2_mixer-2.0.0
 
+#mixer1?
+sudo apt-get install libsdl-mixer1.2-dev
```

Comment:
Sorry, I had this in here because it was asking for mixer1.2, but then I realized it was asking for it because it was trying to use it - which ties into the big bug I had before we had on messenger chat.

I'll remove this libsdl-mixer1.2

Reply:
So we are using sdl_mixer2?

Reply:
yep